README.md

```diff
@@ -374,13 +374,14 @@ cacert_content: |
 ```
 ## Python compatibility
 This library is tested with the following versions of Python:
-* Python 2.7
+* Python 2.7 (last version 0.10.0)
```

Comment:
I just tagged a `0.11.0` version, is it ok to update it here?

Reply:
Done